flight/test_files/waypoint_pathfinding_test.py

```diff
@@ -214,6 +269,10 @@ def main() -> None:
         )
         boundary_vertices.append(Point(easting, northing))
 
+    # Generate random waypoints to use for the waypoint state unit test
+    # Prints json data to be pasted in ../data/waypoint_data.json
+    print(generate_random_waypoints(boundary_vertices, force_zone_number, force_zone_letter))
```

Comment:
Do we really want to print all 100 randomly generated waypoints?

Reply:
I can change it if necessary. This was just a quick way to generate random waypoints for the waypoint state unit test. I would run the waypoint pathfinding test (which only tests pathfinding) and then copy the printed waypoints into waypoint_data.json.